.github/workflows/build.yml

```diff
@@ -22,16 +22,12 @@ env:
   MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
   MAVEN_CLI_OPTS: >-
     --settings settings.xml
-    -fae -U -B -V
-    -Dspring.main.banner-mode=off
-    -Ddoclint=none
-    -DaltReleaseDeploymentRepository=alfresco-public::default::https://artifacts.alfresco.com/nexus/content/repositories/releases
-    -DaltSnapshotDeploymentRepository=alfresco-public-snapshots::default::https://artifacts.alfresco.com/nexus/content/repositories/snapshots
+    -fae -U -V --no-transfer-progress -Dspring.main.banner-mode=off -Ddoclint=none
+    -DdeployAtEnd=true
```

Comment:
This would be better inside the pom.xml configuration maybe (?).
With a maven-deploy-plugin configuration similar to https://github.com/Alfresco/alfresco-process/blob/develop/pom.xml#L300

Reply:
Fixed